### PR TITLE
Add Let's Data Podcast on Podcasts pt_BR

### PR DESCRIPTION
**Pull request by lucassmacedo**
Add Let's Data Podcast on Podcasts pt_BR

## What does this PR do?
Add Let's Data Podcast on Podcasts pt_BR

### Description
O Let's Data Podcast traz pessoas interessantes para conversar sobre ciência de dados e inteligência articificial e outras aleatoriedades da vida. Criado e apresentado por Bernardo Lago, Felipe Schiavon e Leon Silva, com o objetivo de divulgar conhecimento aliando dois enfoques: técnico e humano. Os episódios são transmitidos ao vivo pelo canal no YouTube. Junte-se a nós e venha aprender mais sobre ciência de dados e IA!

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: casts/free-podcasts-screencasts-pt_BR.md [hunk 1]
```diff
@@ -33,8 +33,10 @@
 
 * [Data Hackers](https://datahackers.com.br/podcast) (podcast)
 * [Data Science Academy](http://datascienceacademy.com.br/blog/podcast-dsa/) (podcast)
+* [Let's Data](https://www.youtube.com/channel/UCS58303Tmd0wIKTddmRskdQ) (podcast)
 * [Pizza de Dados](https://pizzadedados.com) (podcast)
 * [Programação Dinâmica - Machine Learning em Python](https://www.youtube.com/playlist?list=PL5TJqBvpXQv5CBxLkdqmou_86syFK7U3Q) (screencast)
+* [Teste de Turing](https://castbox.fm/channel/Teste-de-Turing-id2546538) (podcast)
```
Comment (davorpa) [inline casts/free-podcasts-screencasts-pt_BR.md:39]: - Complete with author to satisfy licensing
- Better multi broadcast platforms like `anchor.fm`. They have various target sources linking to common ones.

```suggestion
* [Teste de Turing](https://anchor.fm/testedeturing) - Erick Fonseca (podcast)
```

**Review by davorpa — CHANGES_REQUESTED** [review]
It's a good moment to apply some changes. When done, request for a review again using GitHub :arrows_clockwise: button. Meanwhile, draft status can be used to work on it if you want.

File: casts/free-podcasts-screencasts-pt_BR.md [hunk 2]
```diff
@@ -33,8 +33,10 @@
 
 * [Data Hackers](https://datahackers.com.br/podcast) (podcast)
 * [Data Science Academy](http://datascienceacademy.com.br/blog/podcast-dsa/) (podcast)
+* [Let's Data](https://www.youtube.com/channel/UCS58303Tmd0wIKTddmRskdQ) (podcast)
```
[the remark this answers is not in the document]
Comment (davorpa) [reply casts/free-podcasts-screencasts-pt_BR.md:36]: What we think about add podcast team authoring?

Bernardo Lago, Felipe Schiavon e Leon Silva

Considere use another platform where we can select various sources. It's sure can be found in their website

[the remark this answers is not in the document]
Comment (lucassmacedo) [reply casts/free-podcasts-screencasts-pt_BR.md:36]: Nice, i think it's good idea

[the remark this answers is not in the document]
Comment (davorpa) [reply casts/free-podcasts-screencasts-pt_BR.md:36]:
```suggestion
* [Let's Data](https://www.youtube.com/playlist?list=PLn_z5E4dh_Lj5eogejMxfOiNX3nOhmhmM) - Bernardo Lago, Felipe Schiavon, Leon Silva (screencast)
```

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**